src/views-components/move-to-dialog/move-to-dialog.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import { Dispatch } from "redux";
+import { withDialog } from "../../store/dialog/with-dialog";
+import { dialogActions } from "../../store/dialog/dialog-actions";
+import { MoveTo } from "../../components/move-to-dialog/move-to-dialog";
+import { reduxForm, startSubmit, stopSubmit } from "redux-form";
+
+export const MOVE_TO_DIALOG = 'moveToDialog';
+
+export const openMoveToDialog = () =>
+    (dispatch: Dispatch) => {
+        dispatch(dialogActions.OPEN_DIALOG({ id: MOVE_TO_DIALOG, data: {}}));
+    };
+
+export const [MoveToProjectDialog] = [MoveTo]
```

Comment:
hmm, that's interesting :)

Reply:
What we really exporting here?

Reply:
@kowalczykpawel7 Could you replace mappings with compose fn? I introduced this mappings, but its true that it might not be understandable. Compose can be imported from redux or lodash.
```
export const MoveToProjectDialog = compose(
withDialog(...),
reduxForm(...)
)(MoveTo)
```

Reply:
@kowalczykpawel7 Yes, the better idea would be to use compose function rather than mappings I introduced. Please, use compose function  for example from redux.